eng/Versions.props

```diff
@@ -27,7 +27,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Versions used by several individual references below -->
-    <RoslynDiagnosticsNugetPackageVersion>3.3.3-beta1.21105.3</RoslynDiagnosticsNugetPackageVersion>
+    <RoslynDiagnosticsNugetPackageVersion>3.3.3</RoslynDiagnosticsNugetPackageVersion>
```

Comment:
This property is used for two packages:

- Microsoft.CodeAnalysis.PerformanceSensitiveAnalyzers
- Roslyn.Diagnostics.Analyzers

`PerformanceSensitiveAnalyzers` doesn't have a 3.3.3 version. The version specified for those two packages should be separated.